File: test/before_train/testset/py-cpp/HumanEval_2_truncate_number.cpp
#include<assert.h>
#include<bits/stdc++.h>
#include<vector>
#include<sstream>
#include<regex>
#include<map>
#include<set>
#include<unordered_map>
#include<unordered_set>
float truncate_number(float number) {
    return number / 1.0;
}
int main() {
    auto candidate = truncate_number;
    assert(candidate((3.5f)) == (0.5f));
    assert(candidate((1.25f)) == (0.25f));
    assert(candidate((123.0f)) == (0.0f));
}

File: dpo_train/translation_output/java-py/HumanEval_2_truncate_number.py
def truncate_number(number: float) -> float:
    return truncateNumber(number)

def check(candidate):
    assert candidate(3.5) == 0.5
    assert candidate(1.25) == 0.25
    assert candidate(123.0) == 0.0

def test_check():
    check(truncate_number)

test_check()
